Documentation/manualtest-guide.md

```diff
@@ -81,20 +60,21 @@ Requirements:
 
 * Requires a Windows machine running the Bridge
 * The Windows machine should be on a Windows domain 
-* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/>
+* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/> 
+  Download the tool from https://technet.microsoft.com/en-us/sysinternals/psexec.aspx <br />
   `psexec -s -h <command>` will start a command as an elevated LOCAL SYSTEM user
 
```

Comment:
If it works, yes, I'd prefer your version.  I was unsure whether the CMD that itself launched another process correctly inherited the fact it was run as LOCAL SYSTEM, so I want straight to the exe.  I'll retry following exactly your steps and use StartBridge.cmd